### PR TITLE
Fix/school 153/dev refactoring

### PR DESCRIPTION
**Pull request by mzavarka**

### CODE REVIEW

**Review by Ivan228777 — APPROVED** [review]
Great!

File: open_schools_platform/user_management/authentication/apis.py [hunk 1]
```diff
@@ -19,7 +20,6 @@
 from ..users.serializers import UserSerializer
 from ..users.services import set_new_password_for_user
 from ...common.services import model_update
-from ...errors.services import NotAcceptableException
```
Comment (Ivan228777) [inline open_schools_platform/user_management/authentication/apis.py:22]: Maybe it is better to write whole path to imported object?

File: open_schools_platform/common/filters.py [hunk 2]
```diff
@@ -3,31 +3,50 @@
...
+        * This feature works only if your input dictionary has  pair [OR_SEARCH_FIELD: some_value]
+    2. Will raise ValidationError when filter get not valid data
+    3. Will order result by "-created_at" field
+        * To use this class your input model type should inherit BaseModel
+        otherwise you can redefine ORDER_DATE_TIME
+        * To disable this feature write ORDER_DATE_TIME=None
+        * Note: symbol '-' is the reverse trigger
+    """
+    OR_SEARCH_FIELD = "search"
+    ORDER_DATE_TIME = "-created_at"
```
Comment (IlyaBat9) [inline open_schools_platform/common/filters.py:22]: Date with a girl?

File: open_schools_platform/user_management/users/constants.py [hunk 3]
```diff
@@ -8,4 +8,4 @@ class RegistrationConstants:
 
 class GenerateConstants:
     ALPHABET = string.ascii_letters + string.digits
-    PASSWORD_LENGTH = 10
+    PASSWORD_LENGTH = 5
```
Comment (IlyaBat9) [inline open_schools_platform/user_management/users/constants.py:11]: In open_schools_platform/user_management/users/serializers.py (51 str) writes that minimum length is equal 6.

File: open_schools_platform/user_management/users/tests/views/test_users_exceptions.py [hunk 4]
```diff
@@ -148,4 +123,4 @@ def test_sms_cannot_be_resend(self):
             "recaptcha": "123456"
         }
         response = self.client.post(self.sms_resend_url(token.key), data)
-        self.assertEqual(400, response.status_code)
+        self.assertEqual(500, response.status_code)
```
Comment (IlyaBat9) [inline open_schools_platform/user_management/users/tests/views/test_users_exceptions.py:126]: 500 is not for application error. This is host error.

**Review by IlyaBat9 — CHANGES_REQUESTED** [review]
This is cool PR.

**Review by IlyaBat9 — APPROVED** [review]

**Pull request merged**